seleniumJSEnv/src/main/scala/org/scalajs/jsenv/selenium/SeleniumComJSRunner.scala

```diff
@@ -37,12 +41,18 @@ class SeleniumComJSRunner(browserProvider: SeleniumBrowser,
   }
 
   def send(msg: String): Unit = {
+    if (comClosed)
+      throw new ComJSEnv.ComClosedException
     awaitForBrowser()
-    browser.getWebDriver.executeAsyncScript(sendScript, msg)
+    val encodedMsg =
+      msg.replace("\u0007", "\u0007\u0007").replace('\u0000', '\u0007')
```

Comment:
I believe that currently there is no issue with `msg.contains("\u0000\u0000")`, but only because of the order of the operations (quite fragile). I will use the `msg.replace("\u0007", "\u0007\u0007").replace("\u0000", "\u00070")` which will also simplify a lot the decoding in JS.